src/openforms/logging/tests/test_admin.py

```diff
@@ -50,7 +54,14 @@ def test_superuser_cant_delete_logs_with_bulk_action(self):
         html_form = changelist.forms["changelist-form"]
 
         # It is not possible to select the "delete_selected" option
-        self.assertIsNone(html_form.fields.get("action"))
+        self.assertEqual(
+            [
+                option
+                for option in html_form.fields.get("action")[0].options
+                if option[0] == "delete_selected"
+            ],
+            [],
+        )
```

Comment:
can't we just assert what *is* present?

```py
options = {value for value, *_ in html_form.fields.get("action")[0].options}
self.assertEqual(options, {"_export"})  # or whatever the relevant value is
```

Reply:
This test is originally to assert that you can't delete records. With your assertion suggestion the test would break whenever a new action gets added? Probably not going happen anyway :thinking:

Reply:
yeah and if the value of the action option in django changes, your version will break and be harder to debug, while this proposed change is a lot more readable and obvious

the previous test assertion worked because it checked that the action dropdown as a whole is absent :)